Sources/swift-bundler/Bundler/XcodeprojConverter/XcodeprojConverter.swift

```diff
@@ -209,6 +210,12 @@ enum XcodeprojConverter {
           log.warning("Could not find target iOS version, assuming 15.0")
         }
 
+        // visionOS deployment version doesn't always seem to be included, so we can just guess if visionOS is supported
+        if visionOSDeploymentVersion == nil, buildSettings?["TARGETED_DEVICE_FAMILY"] != nil {
```

Comment:
Just a reminder to me to test that xcodeproj converting works for visionOS projects